types/database/raw_values.d.ts

```diff
@@ -93,11 +93,16 @@ type TermsOfService = {
     text: string;
 };
 
+type ChannelMembership = {
+    channel_id: string;
+    user_id: string;
+}
+
 type RawValue =
   | AppInfo
   | Channel
   | ChannelInfo
-  | ChannelMembership
+  | MyChannelMembership
```

Comment:
The distinction with MyChannelMembership and ChannelMembership is because we only use the relationship "user-channel" for memberships. More information is brought from the API, but most of that information should not being sent in the first place (https://mattermost.atlassian.net/browse/MM-40440).

Reply:
huh?? we can have channelMembership to add another user as a member of the channel for other purposes, like channel management, this change makes it a bit weird